.github/workflows/mocha.yml

```diff
@@ -37,19 +37,8 @@ jobs:
       - run: npm install --production
       - run: npm run test:smoke
 
-  eslint:
-    name: ESLint Check
-    runs-on: ubuntu-latest
-    needs: smoke
-    steps:
-      - uses: actions/checkout@v2
-      - uses: bradennapier/eslint-plus-action@v3.4.2
-        with:
-          issueSummary: false
-          npmInstall: false
-
-  markdown:
-    name: Markdown Check
+  lint:
+    name: Linting code and markdown
     runs-on: ubuntu-latest
     needs: smoke
     steps:
```

Comment:
There is a v2 release of `actions/setup-node`, should we upgrade?

Reply:
I checked `actions/setup-node` README. I believe the most helpful change is downloading node from node.js release cache.
I don't know how effective we can get. I'll check it separately from this issue.